src/components/SimpleRow.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from "react";
+
+import "../styles/SimpleRow.css";
+import constApi from "../const/const";
+
+class SimpleRow extends Component {
+  render = () => {
+    let rowData = this.props.rowData;
```

Comment:
propTypes

Reply:
+ use const instead of let and also use ES6 destructuring
so it will be:
const { rowData } = this.props;